dali/operators/random/noise/salt_and_pepper_noise.cc

```diff
@@ -28,7 +28,7 @@ The shape and data type of the output will match the input.
     .AddOptionalArg<float>("prob",
       R"code(Probability of an output value to take a salt or pepper value.)code",
       0.05f, true)
-    .AddOptionalArg<float>("salt_to_pepper_prob",
+    .AddOptionalArg<float>("salt_vs_pepper",
       R"code(Probability of a corrupted output value to take a salt value.)code",
```

Comment:
```suggestion
      R"code(Ratio of a corrupted output value to take a salt value.)code",
```

Reply:
In any case it'd be "Ratio of corrupted output values that will take the salt value". But anyway, it is a probability. Why would we want to change it?

Reply:
It is the probability of getting a salt value (and not a pepper value), given that this is a corrupted value.

Reply:
It's definitely not a ratio. Ratio of 1 would mean that they are equal - and it would be impossible to get only salt (the ratio would need to be infinite).

Reply:
OK